docs/source/community/community-call-notes/2021-january.md

```diff
@@ -0,0 +1,117 @@
+
+* **Nick Bollweg** [jupyterlab-lsp 3.2](https://pypi.org/project/jupyterlab-lsp/) on `pypi` & `conda-forge`. JupyterLab
+:three:. :eight: _[Language Client](https://langserver.org)_ features. Ju(lia)+Py(thon)+R _Language Servers_+_Kernel_+_Notebook_, 
+plus [more](https://jupyterlab-lsp.readthedocs.io/en/latest/Language%20Servers.html). [Pre-JEP Issue](https://github.com/jupyter/enhancement-proposals/issues/67)
+to move towards an official sub-project 🚀.
+* **Nick Bollweg** (_or someone better :muscle:_) [JupyterLab 2.3.0rc0](https://github.com/jupyterlab/jupyterlab/blame/2.3.x/docs/source/getting_started/changelog.rst#L6-L12)
+is a performance-focused prerelease, on `pypi`/`conda-forge`, these features will land in 3.1. Ready for
+[testing](https://github.com/robots-from-jupyter/robotframework-jupyterlibrary/pull/34)!
+* **Frédéric Collonval**: Leverage JupyterLab modularity to customize the UI with a alternative 
+[launcher](https://github.com/jupyterlab-contrib/jlab-enhanced-launcher) and a [cell toolbar](https://github.com/jupyterlab-contrib/jlab-enhanced-cell-toolbar) - demonstrate easier distribution thanks to JLab3.
+* **Corentin Cadiou**: presentation of [ipysphaghetti](https://github.com/cphyc/node_editor) (name not settled yet) a JLab 
+(3+) extension implementing a node-based approach to interact with your data.
+* **Thorin Tabor:** Notebook Projects, a mechanism for encapsulating multiple user environments in a JupyterHub instance
+
+## Other Links Shared
+
+- Take the [Jupyter survey](https://blog.jupyter.org/survey-jupyterlab-and-beyond-88c7fbd27a79)!
+- some nteract links after @raman's pre-demo:
+  - https://docs.nteract.io/getting-started-with-nteract/
+  - https://github.com/nteract/nteract
+- semi-related "instant-on" compute/notebooks
+  - [jyve](https://deathbeds.github.io/jyve/lab/), static JupyterLab build
+  - [pyodide](https://github.com/iodide-project/pyodide), full CPython in the browser
+- hackmd's FOSS flavor is now [HedgeDoc](https://github.com/hedgedoc/hedgedoc)
+  - we need a Hub/Lab Extension :hedgehog:
+- "run cells with tags"
+  - from `cell-toolbar` demo
+  - could this be supported in multiple clients, e.g. _JupyterLab Notebook_ UI and `nbclient`/`nbconvert` CLI
+- workflows with (some) Jupyter integration
+  - [cylc](https://github.com/cylc/cylc-ui/issues/90)
+  - [dask](https://docs.dask.org/en/latest/graphs.html)
+  - [dagster](https://github.com/dagster-io/dagster) ([binder](https://gist.github.com/bollwyvl/0fe7a6f89251992ab1a542ac2b4051c4))
+  - [openmdao](https://github.com/OpenMDAO/OpenMDAO)
+  - [prefect](https://stories.dask.org/en/latest/prefect-workflows.html)
+- [GenePattern](https://github.com/genepattern) and [GenePatternNotebook](https://notebook.genepattern.org/#gsc.tab=0)
+- [Integrative Genetics Viewer](http://software.broadinstitute.org/software/igv/) and their [Jupyter extension](https://github.com/igvteam/igv-jupyter)
+
+
+## Q/A
+
+> **As a company, how do we start contributing?**
```

Comment:
this should 💯 be documented somewhere, can we find a way to pull this out and turn it into a blog post, or Discourse post, or something like that?

Reply:
That's a great point! It would need to be cleaned up a little, but I think it could be a blog or Discourse post easily. Discourse might be good because it allows for more conversation so people can add and ask questions as wanted, but I also know things can get lost in there. 

I think it might also make sense to include it in the [Jupyter documentation for contributing](https://jupyter.readthedocs.io/en/latest/contributing/content-contributor.html) since that seems like the purpose of the section. I'd just add relevant (not 4.0 specific) points in sections that already exist. This is the contributing help that is linked from [jupyter.org](https://jupyter.org/documentation), so I think it's likely people might go looking there.

Thoughts?

Reply:
I like the idea of starting out with the Discourse, seeing how the conversation evolves from there, and then potentially putting it into "official" docs somewhere